wlauto/workloads/googlephotos/uiauto/src/com/arm/wlauto/uiauto/UiAutomation.java

```diff
@@ -162,26 +165,22 @@ public void selectWorkingGallery() throws Exception {
         if (discovered) {
             workdir.clickAndWaitForNewWindow();
         } else {
-            throw new UiObjectNotFoundException("Could not find \"wa-working\" folder");
+            throw new UiObjectNotFoundException("Could not find folder : " + directory);
         }
-    }
 
-    // Helper to click on an individual photograph based on index in wa-working gallery.
-    public void selectPhoto(final int index) throws Exception {
```

Comment:
Since there is now only ever 1 image in these subfolders, select the image at index 1 (or 0 on some devices)